src/Engine/ParselessPhraseDB.cpp

```diff
@@ -36,6 +36,7 @@ ParselessPhraseDB::ParselessPhraseDB(
     assert(buf != nullptr);
     assert(length > 0);
 
+    // NOLINTBEGIN(readability-magic-numbers)
```

Comment:
Feel free to remove lines 46-51 (from `uint32_t x = ...` to `assert (x == ...)`, so that you can remove the magic number suppression. These lines have the same effect as lines 41-44 as a way to sanity-check the header, and are actually redundant.

Reply:
fixed in https://github.com/openvanilla/fcitx5-mcbopomofo/pull/84/commits/fe162788bb678085edc195ab09e40013eea0f95e